sources/editor/Stride.GameStudio.Avalonia/Views/MainView.axaml

```diff
@@ -6,6 +6,7 @@
              xmlns:ctrl="using:Stride.Core.Assets.Editor.Avalonia.Controls"
              xmlns:vm="using:Stride.GameStudio.Avalonia.ViewModels"
              xmlns:vw="using:Stride.GameStudio.Avalonia.Views"
+             xmlns:gh="clr-namespace:Stride.GameStudio.Avalonia.Helpers"
```

Comment:
nit: prefer the `using` syntax. `clr-namespace` is the old WPF way.

Reply:
oh, Didn't know that ! I made fix now